### DISCUSSION TITLE
☺️ Which do you prefer, Osaka-style or Hiroshima-style okonomiyaki? ☺️

### DISCUSSION THREAD

**Discussion by nilnano** [NAIST-SE/AttractingContributors#838] [2023 Attraction Contributions]
☺️ Please tell me which do you prefer, Osaka-style or Hiroshima-style okonomiyaki? ☺️

https://github.com/nilnano/AttractingContributors2/blob/main/README.md
https://github.com/nilnano/AttractingContributors2/blob/main/README.md
https://github.com/nilnano/AttractingContributors2/blob/main/README.md
https://github.com/nilnano/AttractingContributors2/blob/main/README.md
https://github.com/nilnano/AttractingContributors2/blob/main/README.md

**Comment by oto10** [comment]
I added my favorite okonomiyaki style to your list.
Could you please add your favorite music to my README.md?
https://github.com/oto10/AttractingContributors-your-favorite-music
Thank you in advance.
